components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/LifeCycleUtils.java

```diff
@@ -393,12 +394,12 @@ private static void addLCStateChangeInDatabase(String user, ApiTypeWrapper apiTy
      */
     private static void sendLCStateChangeNotification(String apiName, String apiType, String apiContext, String apiVersion,
                                                       String targetStatus, String provider, int apiOrApiProductId,
-                                                      String uuid, String organization, String securityScheme) throws
+                                                      String uuid, String organization, String securityScheme, String action, String currentStatus, String apiVisibility) throws
```

Comment:
Fix the indentation

Reply:
Fixed with f9052eea1b85d6e35e35bfb0c5a8ba69304cba05